src/main/scala/modbat/mbt/Modbat.scala

```diff
@@ -381,7 +386,63 @@ class Modbat(val mbt: MBT) {
       mbt.log.info(
         modelStr + nCoveredTrans + " transitions covered (" +
           nCoveredTrans * 100 / nTrans + " % out of " + nTrans + ").")
+
```

Comment:
This function is already quite long; please refactor your code into an extra small function that you call from this place in `coverage`.